arb_os/gasAccounting.mini

```diff
@@ -1181,10 +1210,20 @@ public func txFeeStats_zero() -> TxFeeStats {
     };
 }
 
-public func txFeeStats_setPrices(stats: TxFeeStats, prices: GasPrices) -> TxFeeStats {
+public impure func txFeeStats_setPrices(stats: TxFeeStats, prices: GasPrices, aggregator: option<address>) -> TxFeeStats {
```

Comment:
Unclear why this became impure?

Reply:
It's impure because it looks up the tx base fee for an aggregator address, which is stored in that aggregator's account in the global account table.